dwave/optimization/_model.pyx

```diff
@@ -93,17 +93,39 @@ cdef class _Graph:
         return value
 
     def decision_state_size(self):
-        r"""An estimated size, in bytes, of the model's decision states.
+        """Return an estimate of the size, in bytes, of a model's decision states.
```

Comment:
```suggestion
        r"""Return an estimate of the size, in bytes, of a model's decision states.
```

Currently you get:
![image](https://github.com/user-attachments/assets/20d148af-0365-4391-a609-cc0aaca08aa0)

With this you'll get:
![image](https://github.com/user-attachments/assets/157bfa03-c445-407e-abc2-c7c30328c0c7)